lahja/_snappy.py

```diff
@@ -0,0 +1,7 @@
+is_snappy_available: bool
+
+try:
```

Comment:
whats the reason for not making this a function that just returns `True` or `False`?